employees/pom.xml

```diff
@@ -47,7 +47,7 @@
         </dependency>
         <dependency>
             <groupId>org.wildfly.swarm</groupId>
-            <artifactId>jaxrs-jaxb</artifactId>
+            <artifactId>jaeger</artifactId>
```

Comment:
Why is `jaxrs-jaxb` module removed? IIRC this was needed for POJO -> XML translation.

Reply:
I think JAXB is now available via the jaxrs fraction (POJO->XML is working)